clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -859,6 +859,8 @@ public void run() {
         sendPushJobDetailsToController();
         closeVeniceWriter();
       } catch (Exception ex) {
+        sendPushJobDetailsToController();
+        closeVeniceWriter();
```

Comment:
How about moving 859-863 to the final block if we have to invoke them anyways?

Reply:
Can we veify that the colo status is being constructed correctly? I don't see how adding more `sendPushJobDetailsToController()` will fix the issue of incomplete colo statuses